client/src/components/J40Alert/index.tsx

```diff
@@ -3,7 +3,12 @@ import {useIntl} from 'gatsby-plugin-intl';
 import {defineMessages} from 'react-intl';
 import * as styles from './j40Alert.module.scss';
 
-const J40Alert = () => {
+interface IJ40AlertProps {
+  location: Location;
+}
+
+const J40Alert = ({location}:IJ40AlertProps) => {
+  const isPadded = location?.pathname.match(/cejst\/?/);
```

Comment:
We should avoid relying on `location` in this way - components should be independent of their context of use, and the mapping from location to page is not guaranteed to be persistent / introduces a hidden dependency. 

If a given page needs to do this just on the page, we should introduce a wrapper and include this as part of the styling of the page

Reply:
Good call! replacing this with an inversion of control pattern allowing the instance of the component determine its styling.